scripts/postinstall.js

```diff
@@ -71,7 +72,7 @@ const fetchAndParseCheckSumFile = async (packageJson) => {
 
   // Fetch the checksum file
   console.info("Downloading", checksumFileUrl);
-  const response = await fetch(checksumFileUrl);
+  const response = await fetch(checksumFileUrl, {agent: new ProxyAgent()});
```

Comment:
Thanks for the contrib. We need to pass down the ProxyAgent to all places where fetch is called. See https://github.com/supabase/cli/pull/1451/files#diff-8d291e232c16ef847e6e526d55be7a3c2ec355821af7da7b948077163769eb70L126

Reply:
Oops, I inadvertently forgot to do that. Adding.